images/linux/scripts/installers/php.sh

```diff
@@ -66,7 +66,6 @@ for version in $php_versions; do
 
     if [[ $version == "5.6" || $version == "7.0" || $version == "7.1" ]]; then
         apt-fast install -y --no-install-recommends php$version-mcrypt php$version-recode
-        apt-get remove --purge -yq php$version-dev
```

Comment:
Hm, interesting, why php-dev library was removed initially...
I don't have objections to add it back but would like to understand the context.

@alepauly, @shadyysf  do you have any context?

Reply:
@maxim-lobanov No idea, looks like it's been like that for as long as I can tell. Is there a size issue perhaps? Do we want php-dev for all those versions?

Reply:
@alepauly PHP has 2 constants `PHP_API_VERSION` and `ZEND_EXTENSION_API_NO` which are bumped for each minor version and makes extensions between minor versions incompatible. Thus to build extensions for say PHP 7.2 one would require `php7.2-dev` which has the `phpize` and `php-config` scripts having correct constants for PHP 7.2.

Reply:
thanks @shivammathur, if you all think we should include `-dev`, I'm good with that.